### ISSUE TITLE
List featural specification in addition to list of segments

### ISSUE THREAD

**Issue by kchall** [PhonologicalCorpusTools/CorpusTools#449]
At least when segments are selected by specifying features, it should be possible to list the environment / selection in terms of those original features in addition to the actual list of segments.

**Comment by jsmackie** [comment]
It turned out to be really trick to do this within one window, so I had to split the task up. The way it works now, when you are choosing segments for an environment, you get a dropdown menu with options "Add segments" and "Add features". 

If you choose "Add features", then you get an inventory window where you cannot select any individual segments. You can only type in feature names. These name will appear in the GUI, but underlyingly a list of segments is passed to the analysis function, so nothing changes there.

If you choose "Add segments", then you get the inventory window where you can either select segments individually, or by feature. However, the GUI will display a list of segments, regardless of which method you use.

In short, both windows do exactly the same thing, it's just a matter of how you want to see things displayed. I will add a note about this to the documentation.

**Comment by jsmackie** [comment]
Docs are updated.
